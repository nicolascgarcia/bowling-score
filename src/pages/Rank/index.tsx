export default function Rank() {
	return (
		<div>
			<h1>Rank</h1>
		</div>
	);
}
