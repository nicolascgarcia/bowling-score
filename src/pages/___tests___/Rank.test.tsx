/* eslint-disable import/no-extraneous-dependencies */
import renderer from 'react-test-renderer';
import Rank from '../Rank';

describe('<Rank/>', () => {
	it('should render Rank', () => {
		const component = renderer.create(<Rank />);
		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
