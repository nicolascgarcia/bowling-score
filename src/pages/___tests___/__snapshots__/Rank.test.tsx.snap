// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Rank/> should render Rank 1`] = `
<div>
  <h1>
    Rank
  </h1>
</div>
`;
